mautic/mautic
# Fix from_name and from_email labels

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      |  
| Related developer documentation PR URL | 
| Issue(s) addressed                     | Fixes #14040 



## Description

When sending a Quick Email to a contact, the labels for the first two input boxes, for lead_quickemail[fromaddress] and lead_quickemail[from] are confusing. They are labeled "From Address" (where it should say From Name), and "From" where it should say "From Address".






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Go to Contacts
3. Open a contact
4. Click the "Send Email" button
5. In the form opened the labels for the first two fields are wrong. They should be From name (not From address) and From address (not From)
